server/application/application.proto

```diff
@@ -65,6 +65,7 @@ message ApplicationUpdateRequest {
 message ApplicationDeleteRequest {
 	required string name = 1;
 	optional bool cascade = 2;
+	optional bool noPrompt = 3;
```

Comment:
This feature shouldn't require backend changes, since the conformation should be done clientside (both in UI and CLI).

Reply:
Done.